PrebidMobile/PrebidMobileRendering/Utilities/PBMUserAgentService.m

```diff
@@ -30,7 +30,7 @@
 
 @interface PBMUserAgentService()
 
-@property (nonatomic, strong) WKWebView *webView;
+@property (nonatomic, strong) UIWebView *webView;
```

Comment:
We can't use UIWebView. It is deprecated. It's usage may block the submission of the app into App Store.